_sass/pages/_partners.scss

```diff
@@ -147,3 +147,16 @@
   background-color: color('primary-lightest');
   border-radius: radius('md');
 }
+
+.roadmap-body {
+  .text, .image {
+    display: inline-block;
+  }
+  .text {
+    padding-right: 50px;
+  }
+  .image {
+    float: right;
+    padding-bottom: 25px;
+  }
+}
```

Comment:
I believe you can use utility classes rather than adding to the sass, see here: https://designsystem.digital.gov/utilities/

Reply:
Yeah looking at the design, I think ideally you could skip almost all of these CSS styles and use a few utility classes like `float-right`,  `margin-left-6`, etc.

May be fine for a quick follow-on pull request if there's a time crunch.

Reply:
Made changes to use USWDS classes instead of CSS stuffs